examples/applications/wikipedia_principal_eigenvector.py

```diff
@@ -74,8 +72,6 @@
 # #############################################################################
 # Loading the redirect files
 
-memory = Memory(cachedir=".")
```

Comment:
this fully removes joblib rather than updating API. Do I miss something?

Reply:
If joblib.Memory is no longer used (see comment below in the diff indicating that it was too slow), I think that we can remove it.